src/assets/i18n/zh.json

```diff
@@ -1 +1 @@
非化学燃料不会受影响","flowRateUnit":"u/秒","pipeFlowRate":"管道流速","pipeFlowRateTooltip":"请输入管道流速","inserterTarget":"机械臂目标","inserterTargetTooltip":"选择机械臂目标","bonuses":"额外加成","miningProductivity":"采矿产能","miningProductivityTooltip":"输入当前采矿产能加成的百分比\n默认 0%","researchSpeed":"研发速度","researchSpeedTooltip":"输入当前研发速度加成\n该设置只影响产品的研发","inserterCapacity":"机械臂运量","inserterCapacityTooltip":"输入机械臂当前运量加成等级\n该设置只影响机械臂的计算","miningSpeed":"采矿速度","miningSpeedTooltip":"输入当前采矿速度加成\n默认 100%","display":"显示","openColumnSettings":"打开列设置","displayRate":"速率显示为","displayRateTooltip":"设置速率使用的时间单位","powerUnit":"电量单位","powerUnitTooltip":"设置列表中电量列要显示的电量单位","helpTranslate":"帮助翻译","language":"语言","languageTooltip":"切换当前显示的语言","theme":"主题","themeTooltip":"选择主题","advanced":"高级","simplexSolver":"单纯形法求解器","simplexSolverTooltip":"选择单纯形法求解器。\nJS BigInt Rational 较慢，但精度高"},"objectives":{"title":"目标","products":"产物","removeProductTooltip":"移除产物","itemTooltip":"选择产物","rateTooltip":"设置生产速度","rateTypeTooltip":"改变速率类型","viaPlaceholder":"设置限制的步骤","viaTooltip":"通过限制一个步骤的生产速率限制产品的生产","noEnabledRecipes":"没有已启用的匹配配方","producers":"生产线","removeProducerTooltip":"移除生产线","recipeTooltip":"选择配方","countTooltip":"设置生产线数量","product":"产物","addAProduct":"添加一个产物","addAProductTooltip":"选择要生产的产物","producer":"生产线","addAProducer":"添加一个生产线","addAProducerTooltip":"选择要生产的配方","selectDisplayRate":"选择速率显示单位"},"list":{"openColumnSettings":"打开列设置","tree":"树结构","items":"物品","itemsUndoTooltip":"取消所有对物品的隐藏","belts":"传送带","beltsUndoTooltip":"取消所有对传送带的修改","wagons":"车厢","wagonsUndoTooltip":"取消所有对车厢的修改","factory":"工厂","factoriesUndoTooltip":"取消所有对工厂的修改","beacons":"插件","beaconsUndoTooltip":"取消所有对插件的修改","power":"电量","pollution":"污染","downloadTooltip":"保存列表为 csv 文件","showDetails":"展开详情","hideDetails":"隐藏详情","overclockTooltip":"超频百分比","beaconEachTooltip":"每工厂插件数","beaconTotalTooltip":"总插件数","openNewTabTooltip":"在新标签页中打开","stepUndoTooltip":"撤销此步骤之后的更改","sources":"原料","destinations":"成品","depletion":"消耗","inputs":"输入","outputs":"输出","inserterCountInfo":"* 机械臂是根据设置以及数据计算出的，数据来自 0.18 版本的实验性数据，\n详情查看 ","defaultRecipe":"默认配方","defaultRecipeInfo1":"点击设置该物品的默认配方。\n如果设置了该选项，那么该选项将作为制造这项物品的唯一配方。","defaultRecipeInfo2":"该配方必须启用并且必须使用这个配方来制造这项物品。","defaultRecipeInfo3":"检测到循环引用，该默认值被忽略。","enabledRecipes":"启用配方","enabledRecipesInfo":"点击配方来启用或是禁用它们\n禁用配方的话将不会使用它来进行计算","total":"总计"},"matrix":{"statistics":"统计数据","simplexResult":"单纯形法求解结果","runtime":"消耗时间","pivots":"主元数","size":"大小","sizeValue":"{{ rows }} 行，{{ cols }} 列","costModifiers":"成本修改","costUndoTooltip":"取消所有成本修改","recipeCostTooltip":"将所有有既定成本的配方（通常是采集）的成本乘以此系数","recipe":"配方","factoryCostTooltip":"将所有在工厂中制作的配方的成本乘以此系数","factory":"工厂","inputCostTooltip":"非配方产生的外部输入物品的成本","input":"输入的物品","ignoredCostTooltip":"被指定为已忽略的物品的成本","ignored":"忽略的物品","tableauPrecisionNote":"注：此表格使用物品列表的数据精度设置","simplexCanonicalTableau":"规范单纯形表","items":"每秒增减个数","producers":"生产线","recipes":"配方","inputs":"物品输入","cost":"成本","objective":"目标","recipeCost":"配方成本","recipeCostUndoTooltip":"取消对配方成本的修改","solution":"解决方案","surplus":"每秒盈余个数"}}
```

Comment:
`"list.factory"` in this file should be `"list.factories"`, the Factories column header isn't translated for this reason

Reply:
That's the only issue I see. Thanks for the contribution!

Reply:
Oh I missed that one. Fixed now.